blockstore/src/main/java/brooklyn/location/blockstore/openstack/OpenstackNewVolumeCustomizer.java

```diff
@@ -0,0 +1,86 @@
+package brooklyn.location.blockstore.openstack;
+
+import brooklyn.location.blockstore.BlockDeviceOptions;
+import brooklyn.location.blockstore.FilesystemOptions;
+import brooklyn.location.blockstore.api.VolumeManager;
+import brooklyn.location.blockstore.ec2.Ec2VolumeManager;
+import com.google.common.base.Optional;
+import com.google.common.collect.Maps;
+import org.apache.brooklyn.location.jclouds.BasicJcloudsLocationCustomizer;
+import org.apache.brooklyn.location.jclouds.JcloudsLocation;
+import org.apache.brooklyn.location.jclouds.JcloudsMachineLocation;
+import org.jclouds.compute.ComputeService;
+import org.jclouds.compute.domain.NodeMetadata;
+
+import java.util.List;
+import java.util.Map;
+
+public class OpenstackNewVolumeCustomizer extends BasicJcloudsLocationCustomizer {
```

Comment:
Sounds like a bug in `$brooklyn:object`. Does it work any better if you override the setters?